src/CMakeLists.txt

```diff
@@ -27,11 +27,12 @@ set(SOURCES
 
 configure_file(
   pumipic_version.hpp.in
-  ${CMAKE_CURRENT_SOURCE_DIR}/pumipic_version.hpp
+  pumipic_version.hpp
```

Comment:
Does `pumipic_version.hpp` get copied to the include directory when `make install` is run?

Reply:
I didn't consider that, it is now fixed.